### PR TITLE
Feat implement storage cleaner in java

### PR DESCRIPTION
**Pull request by Prakhar-Agarwal-byte**
Added a storage cleaner function demo in Java

Makes progress towards appwrite/appwrite#2031

### CODE REVIEW

**Comment by Prakhar-Agarwal-byte** [comment]
@PineappleIOnic Please merge the PR now 😀

**Comment by Prakhar-Agarwal-byte** [comment]
@PineappleIOnic Please merge the PR as today is the last day for claiming appwrite swags😅

File: java/storage-cleaner/src/main/java/StorageCleaner.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    public static void main(String[] args) throws AppwriteException {
+        Client client = new Client()
+                .setEndpoint(System.getenv("APPWRITE_ENDPOINT"))
+                .setProject(System.getenv("APPWRITE_FUNCTION_PROJECT_ID"))
+                .setKey(System.getenv("APPWRITE_API_KEY"));
+        Storage storage = new Storage(client);
+        try {
+           double daystoExpire = Double.parseDouble(System.getenv("DAYS_TO_EXPIRE"));
+            storage.listFiles(
+                    new Continuation<Object>() {
```
Comment (Meldiron) [inline java/storage-cleaner/src/main/java/StorageCleaner.java:26]: Lets add pagination, this will only return 25 files.

**Review by Meldiron — CHANGES_REQUESTED** [review]

File: java/storage-cleaner/src/main/java/StorageCleaner.java (hunk 1 above)
Comment (Prakhar-Agarwal-byte) [reply java/storage-cleaner/src/main/java/StorageCleaner.java:26]: In the API docs it's mentioned that it lists all user files. https://appwrite.io/docs/client/storage#storageListFiles

Comment (christyjacob4) [reply java/storage-cleaner/src/main/java/StorageCleaner.java:26]: @Prakhar-Agarwal-byte yes that endpoint can be used to list all the files. But you need to use the `limit` and `offset`parameters to get all of them.

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Please add support for pagination in your cloud function

**Comment by christyjacob4** [comment]
@Prakhar-Agarwal-byte closing this PR for now. Feel free to reopen it / create a new one when you're ready to start working on it once again

**Pull request closed without merging**